nrf-hal-common/src/saadc.rs

```diff
@@ -1,4 +1,20 @@
 //! HAL interface to the SAADC peripheral.
+//!
+//! example usage:
+//! ```rust
+//! use nrf52840_hal::prelude::*;
```

Comment:
Generally the current style is to merge imports. So you could combine these as such:

```rust
use nrf52840_hal::{
    prelude::*,
    saadc::{SaadcConfig, Saadc},
};
```

As a note, you only need to use the `as` directive when renaming the import. e.g. `use foo::Foo as Foo;` is the same as `use foo::Foo;`. You would use `as` when doing something like:

```rust
use bar::Foo as BarFoo;
use baz::Foo as BazFoo;
```

Reply:
Yup, I've added the `as` to make the following code more concise but just directly including `nrf52840_hal::saadc::SaadcConfig` and `nrf52840_hal::saadc::Saadc` doesn't work because you also need to import the pac's `OneShot` trait which then makes the import statements even less concise. Is there a nicer way to resolve this conundrum? :D